app/server/locale/directory_listing.en.yml

```diff
@@ -0,0 +1,12 @@
+en:
+  business_categories: ['Food',
```

Comment:
these don't need to be in a separate yml file, they can be moved inside of the ussd.yml